packages/server/graphql/mutations/helpers/resolveDowngradeToPersonal.ts

```diff
@@ -19,7 +19,7 @@ const resolveDowngradeToPersonal = async (
     console.log(e)
   }
 
-  const [rethinkResult] = await Promise.all([
+  const [, teamIds] = await Promise.all([
```

Comment:
let's not return `teamIds` here so we can remove the RETURN from our write query.
then, in https://github.com/ParabolInc/parabol/blob/36a79f894e9a78622a4fa95c8aa308591c8f1e6c/packages/server/graphql/mutations/downgradeToPersonal.ts#L44 we can just get the teamIds via the new dataLoader